eisvogel.tex

```diff
@@ -432,7 +432,7 @@
 \subtitle{$subtitle$}
 $endif$
 $if(author)$
-\author{$for(author)$$author$$sep$ \and $endfor$}
+\author{$for(author)$$if(author.name)$$author.name$$if(author.affiliation)$($author.affiliation$)$endif$$else$$author$$endif$ \and $endfor$}
```

Comment:
This doesn't seem right. There should be a space between the `author.name` and the brackets surrounding `author.affiliation`. Currently it renders to `John Doe(University of Nowhere)`. Also you removed the support for the separator `$sep$`. Please take a look at <https://pandoc.org/MANUAL.html#using-variables-in-templates>.